samples/dicom-demo-env/README.md

```diff
@@ -0,0 +1,58 @@
+# Medical Imaging Network Demo Environment
+Getting your medical imaging data to the cloud shouldn't be difficult, and can be done incrementally, safely, and without major disruptions to existing workflows.
+
+This hands-on lab / demo highlights how an organization with existing on-prem radiology infrastructure can take the first steps to intelligently moving their data to the cloud, _without_ disruptions to the current workflow. 
+
+## Overview
+This sample contains the deployment templates to automatically provision a full end-to-end demo environment in an Azure resource group and the full instructions to configure and run the demo.  The deployment includes:
+- A simulated on-prem network containing: 
+    - A PACS (Orthanc)
+    - An X-ray modality (simulated)
+    - A user's personal computer
+- An instance of Azure Health Data Services DICOM service 
+- The medDream zero-footprint viewer, hosted in Azure
+- The Qvera QIE intelligent DICOM router 
+
+## The Setup
+There are a few core steps that need to be accomplished prior to deploying the infrastructure.
+
+### App Registration
+To allow Azure Active Directory to be used as an identity provider to secure access to the DICOM service, you must create an App Registration. The App Registration tells Azure AD about the application that will be speaking to the DICOM service and needs to use Azure AD. In this case, the App Registration will be re-used by two different applications: Qvera QIE and the Softneta medDream viewer. In a production environment, you'll likely want two different app registrations so access can be fine-tuned by application.
+
+To create an App Registration, follow the steps here: [Add App Registration](add-app-reg.md)
+
+You should now have several key pieces of data which will be used in the upcoming steps:
+- Application (client) ID
+- Directory (tenant) ID
+- Client secret
+- Principal Object ID of the App Registration
+
+### Deploying the infrastructure
+Creating the cloud and on-prem environments in Azure can be done in one of two ways. You can use the Bicep command line, which is easiest if you are running this repo multiple times for development. Or you can use the Azure Portal user experience, which is easiest for a one-off deployment. Both create identical environments.
+
+#### Deploy using the Azure Portal
+To use reasonable default values, simply click [![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FStevenBorg%2FRSNA-2022-Demo%2Fmain%2Fenvironments%2Fall-up-demo-deployment.json)
```

Comment:
NIT: sometimes it's best to prompt users to open this in a new tab so they don't lose instructions.  Also, before or after merging we need to change the URL to this repo vs `StevenBorg` for the deploy button.

Reply:
Good call.  I'll change the text now and fix the link after merging.